src/openzaak/utils/help_text.py

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2020 Dimpact
+from django.utils.translation import ugettext_lazy as _
+
+
+def mark_experimental(text):
+    warning_msg = _(
+        "Warning: this feature is experimental and not part of the API standard"
+    )
+    return "{} **{}**".format(text, warning_msg)
```

Comment:
```suggestion
    warning_msg = _(
        "Warning: this feature is experimental and not part of the API standard"
    )
    return "**EXPERIMENTEEL** {}".format(text)
```

I think one word is enough. I saw Steven's suggestion, but it looks a little excessive. @stevenbal What do you think? 

Anyway this is an optional suggestion, no need to implement it if you disagree.

Reply:
That's fine for me, I just wanted a function for it to make it consistent everywhere :+1: